flow/src/org/labkey/flow/controllers/well/chooseGraph.jsp

```diff
@@ -115,7 +116,7 @@
         <% if (hasScripts)
         {
         %>
-        <tr><td>Analysis Script:</td><td><select name="<%=FlowParam.scriptId%>" onchange="this.form.submit()">
+        <tr><td>Analysis Script:</td><td><select name="<%=h(FlowParam.scriptId)%>" onchange="this.form.submit()">

```

Comment:
Should FlowParam implement EnumHasHtmlString? Then h() would be unnecessary.

Reply:
It should!